services/bitbucket/bitbucket-last-commit.service.js

```diff
@@ -0,0 +1,77 @@
+import Joi from 'joi'
+import { age as ageColor } from '../color-formatters.js'
+import { BaseJsonService, NotFound, pathParam, queryParam } from '../index.js'
+import { formatDate } from '../text-formatters.js'
+
+const schema = Joi.object({
+  values: Joi.array().items({
+    date: Joi.string().isoDate().required(),
+  }),
+}).required()
+
+const queryParamSchema = Joi.object({
+  path: Joi.string().uri({ relativeOnly: true }),
```

Comment:
once we merge the other related PR, this could become `relativeUri`, right?

Reply:
Yep!

Reply:
OK. I've just put https://github.com/badges/shields/pull/10041 in the merge queue. Once https://github.com/badges/shields/pull/10041 is in, do you want to update this. Then I'll give it a final check over.

Reply:
https://github.com/badges/shields/pull/10043/commits/610b149f951e7bab29cf4384272650753f820c71